src/stylesheets/styles.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
```

Comment:
Nice job. Remember to make the necessary scripts and change the old ones on Package.json file.
Remember to update the readme to describe the scrips behaviour.